connection.go

```diff
@@ -344,3 +400,89 @@ func (sc *snowflakeConn) stopHeartBeat() {
 	}
 	sc.rest.HeartBeat.stop()
 }
+
+func updateRows(data execResponseData) (int64, error) {
+	var count int64 = 0
+	for i, n := 0, len(data.RowType); i < n; i++ {
+		v, err := strconv.ParseInt(*data.RowSet[0][i], 10, 64)
+		if err != nil {
+			return -1, err
+		}
+		count += v
+	}
+	return count, nil
+}
+
+type childResult struct {
+	id  string
+	typ string
+}
+
+func getChildResults(IDs string, types string) ([]childResult, error) {
+	if IDs == "" {
+		return nil, nil
+	}
+	queryIDs := strings.Split(IDs, ",")
+	resultTypes := strings.Split(types, ",")
+	if len(queryIDs) != len(resultTypes) {
+		return []childResult{}, &SnowflakeError{
+			Number:  ErrNoChildrenReturned,
+			Message: errMsgFailedToRetrieveChild,
+		}
+	}
+	res := make([]childResult, len(queryIDs))
+	for i, id := range queryIDs {
+		res[i] = childResult{id, resultTypes[i]}
+	}
+	return res, nil
+}
+
+func (sc *snowflakeConn) getQueryResult(ctx context.Context, resultPath string) (execResponse, error) {
+	headers := make(map[string]string)
+	headers["Content-Type"] = headerContentTypeApplicationJSON
+	headers["accept"] = headerAcceptTypeApplicationSnowflake // TODO v1.1: change to JSON in case of PUT/GET
```

Comment:
Is this comment valid?

Reply:
The headers were a part of the query fetching function that I copied. I'm not sure if it's valid (presumably it was left as a TODO) but I left it incase someone wanted to revisit the header. I can look into it and keep or remove it. Would that be acceptable?

Reply:
If this function is purely to get result set, you won't need to  change this header for PUT and GET command. This comment is for `query-request` call

Reply:
Got it. I'll remove it